code/stable/dblpendulum/SRS/DblPendulum_SRS.tex

```diff
@@ -729,7 +729,7 @@ \subsubsection{General Definitions}
 Substituting for $\mathbf{I}$
 
 \begin{displaymath}
-m {L_{\text{rod}}}^{2} \frac{\,d\frac{\,d{θ_{p}}}{\,dt}}{\,dt}=-{L_{\text{rod}}} m \mathbf{g} \sin\left({θ_{p}}\right)
+m {L_{\text{rod}}}^{2.0} \frac{\,d\frac{\,d{θ_{p}}}{\,dt}}{\,dt}=-{L_{\text{rod}}} m \mathbf{g} \sin\left({θ_{p}}\right)
```

Comment:
That this is a `2.0` is odd. Exponents are usually / often integers. There should be a real-to-a-natural 'power' function.

Reply:
I was thinking that one of the next tasks might be to add a whole series of functions like this. Though, it seems that Java, C#, and Swift at least typically have their respective "Math.Pow" as Double X Double -> Double

Reply:
For now, I switched it to using `square` which uses the new `exactDbl`, so it will be visually pleasing at least

Reply:
`^` in a number of these languages is what's wanted.